app/src/main/res/values-it/strings.xml

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="welcome_message">Istruzioni:\n1. Inserisci il tuo Livello Allenatore\n2. Premi start e consenti la registrazione schermo\n3. In \'Pokémon Go\' seleziona il Pokémon\n4. Premi il bottone IV per analizzare il Pokémon\n5. Correggi i dati se necessario\n6. Analizza IV!</string>
+    <string name="goiv_is_open_source">GoIV è open source!\ngithub.com/farkam135/GoIV\nreddit.com/r/GoIV</string>
+    <string name="trainer_level">Livello Allenatore</string>
+
+    <string name="pokemon_name">Nome Pokémon:</string>
+    <string name="cp">PL</string>
+    <string name="hp">PS</string>
+    <string name="use_the_slider_below_to_align_the_arc">Usa lo slider per allineare l\'arco</string>
+    <string name="cancel">Cancella</string>
+    <string name="check_iv">Analizza IV</string>
+    <string name="main_not_numeric">%1$s non è un numero.</string>
+    <string name="main_invalide_trainerlvl">Livello Allenatore non valido!</string>
+    <string name="notification_title">GoIV in esecuzione - Livello %1$s</string>
+    <string name="notification_text">Tocca per aprire</string>
+    <string name="ivtext_stats">Att: %1$02d Dif: %2$02d PS: %3$02d — %4$3d%%</string>
```

Comment:
Hm. Have you checked if we use this? I might or might not have seen code hardcoding the English version.